src/tools/dev/scripts/bv_support/bv_adios.sh

```diff
@@ -71,7 +71,7 @@ function bv_adios_info
     export ADIOS_FILE=${ADIOS_FILE:-"adios-${ADIOS_VERSION}.tar.gz"}
     export ADIOS_COMPATIBILITY_VERSION=${ADIOS_COMPATIBILITY_VERSION:-"${ADIOS_VERSION}"}
     export ADIOS_URL=${ADIOS_URL:-"http://users.nccs.gov/~pnorbert"}
-    export ADIOS_BUILD_DIR=${ADIOS_BUILD_DIR:-"adios-${ADIOS_VERSION}"}
+    export ADIOS_SRC_DIR=${ADIOS_SRC_DIR:-"adios-${ADIOS_VERSION}"}
```

Comment:
When the TPL is built in source, the source and build dir are one in the same. So, why not just have two lines hee, one for the SRC dir *and* another for the build dir.

Reply:
Because it would require hand editing 50+ bv files.